evergreen.yml

```
@@ -1318,3 +1364,9 @@ buildvariants:
     run_on: [ubuntu1804-large]
     tasks:
       - name: release
+
+  - name: code-security
+    display_name: "Code Security"
+    run_on: [ubuntu2204-small]
+    tasks:
+      - name: semgrep
```

```suggestion
  - name: code-security
    display_name: "Code Security"
    run_on: [ubuntu2204-small]
    tasks:
      - name: semgrep
      
```
[nit] new line.

Also, since I don't think it's going to merge conflict, I wanted to point out that I merged the `buildvariant` below into main, so you'll need to pull my changes, so we can combine these into the same `buildvariant`. 
```
  - name: mongosqltranslate-code-quality-security
    display_name: "Mongosqltranslate Code Quality and Security"
    run_on: [ubuntu2204-small]
    tasks:
      - name: mongosqltranslate-sbom
```